code/drasil-example/gamephysics/lib/Drasil/GamePhysics/Requirements.hs

```diff
@@ -131,9 +132,4 @@ understandability = cic "understandability" (foldlSent [
   ]) "Understandability" nonFuncReqDom
 
 maintainability :: ConceptInstance
-maintainability = cic "maintainability" (foldlSent [
-  S "If a likely change is made" `S.toThe` S "finished software, it will take", 
-  addPercent (10 :: Integer), S "percent" `S.ofThe` S "original development time,",
-  S "assuming the same development resources are available"
-  ]) "Maintainability" nonFuncReqDom
-
+maintainability = mkMaintainableNFR "maintainability" (10 :: Integer) "Maintainability"
```

Comment:
Do we need to specify `:: Integer` for each of these calls, or can we just pass in `10`; either way, great job! I like this code reuse!

Reply:
Before, it was necessary; however, since the function `mkMaintainableNFR` has an explicit type declaration for that parameter, it is no longer needed. Good catch!